doc/nrf/ug_bootloader_external_flash.rst

```diff
@@ -12,7 +12,7 @@ See :ref:`pm_external_flash` for general information about how to set up partiti
 
 .. note::
 
-   Currently, only MCUboot supports booting images stored in the external memory partition.
+    Currently, only MCUboot supports booting images stored in the external memory partition.
```

Comment:
Why was a space added here? Isn't three spaces the correct amount for a note?

Reply:
Out of habit, I guess. I always indented notes compared with the tag (and rarely or never used the blank line after tag). But I see these notes display well, so I revert.